kubernetes/config.saat.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  # Node specific vars
+  USE_CSRF_PROTECTION: "true"
+  SECURITY_REFERRER_POLICY: "origin"
+  HPKP_MAX_AGE: "2592000"
+  HPKP_SHA256S: "Set-proper-SHA256s"
+  NODE_ENV: "production"
+  UV_THREADPOOL_SIZE: "64"
+  NODE_CONFIG_DIR: "./config"
+  TS_BASE_URL: "./src/main"
+
+  # Logging vars
+  REFORM_TEAM: "ccd"
+  REFORM_SERVICE_NAME: "admin-web"
+  REFORM_ENVIRONMENT: "aat"
+
+  # Application vars
+  # IDAM
+  IDAM_BASE_URL: "https://preprod-idamapi.reform.hmcts.net:3511"
+  IDAM_S2S_URL: "http://rpe-service-auth-provider-saat.service.core-computer-saat.internal"
+  IDAM_ADMIN_WEB_SERVICE_KEY: "${IDAM_SERVICE_KEY}"
+  IDAM_SERVICE_NAME: "ccd_admin"
+  IDAM_LOGOUT_URL: "https://idam.preprod.ccidam.reform.hmcts.net/login/logout"
```

should use IDAM test: 
```suggestion
  IDAM_LOGOUT_URL: "https://idam-test.dev.ccidam.reform.hmcts.net/login/logout"
```